docs/source/upgrade.rst

```diff
@@ -8,6 +8,26 @@ This is a list of changes introduced in each release that users should be aware
 Most changes are carefully designed not to break existing code; however changes that may possibly break them are highlighted with a box.
 
 
+Chainer v5
+==========
+
+Changed Behavior of ``Optimizer.new_epoch``
+-------------------------------------------
+
+This change only affects to a minor number of users (who call ``new_epoch()`` method of Optimizers while using a trainer, or who implement their own Updater class).
+
+Optimizers provide ``new_epoch()`` method, which can be used to change optimizer behavior depending on the current epoch number.
+Prior to Chainer v5, this method was expected to be called by users.
+In Chainer v5, Updaters call ``new_epoch()`` method of Optimizers automatically.
+If you are calling ``new_epoch()`` method manually while using a trainer (Updater), you may need any of the following fixes:
+
+* Pass ``auto_new_epoch=False`` to the constructor of the Updater to stop ``new_epoch()`` being called automatically by Updaters.
+* Avoid calling ``new_epoch()`` method manually.
+
+If you are implementing your own Updater class, you may need to update your code to automatically call ``new_epoch()``.
+See `#4608 <https://github.com/chainer/chainer/pull/4608>`__ for the detailed discussions.
```

Comment:
"... for a more detailed discussion" is more natural.
Btw, actually #4608 does not include any discussion. Is it worth mentioning?

Reply:
There're no discussion, but updater implementors may find it useful to understand how to fix their own updaters.
Changed to:
> See `#4608 <https://github.com/chainer/chainer/pull/4608>`__ for more details.

Reply:
I think the information in that PR is (and should be) written in this documentation.

Reply:
I assume not many people implements own Updaters, so it's rather better to keep it simple.